### PR TITLE
Properly Escape Arguments to Runner

### LINKED ISSUE #192: Su invocation is not escaped properly for TS work [JIRA: TOOLS-167]
When using TS packages, riak-admin is crippled for defining bucket types because of the need to use `su` to the `riak` user.

As seen below from `sh -x`, the JSON is not wrapped properly, and the shell complains.

```
exec su - riak -c '/usr/sbin/riak-admin bucket-type create testone \{\"props\":\{\"table_def\":\"CREATE TABLE testone (field1 varchar not null, field2 varchar not null, time timestamp not null, PRIMARY KEY((field1, field2, quantum(time, 15, s)), field1, field2, time))\"\}\}'
```

### PR DESCRIPTION
**Pull request by gcymbalski**
Fixes #192

### CODE REVIEW

**Comment by gcymbalski** [comment]
This looks to successfully fix the regular expression in question and pass the appropriately-escaped string back out for consumption.

**Comment by ooshlablu** [comment]
+1

**Comment by macintux** [comment]
:+1:

File: priv/base/env.sh [hunk 1]
```diff
@@ -238,8 +238,8 @@ check_user() {
         #     riak-admin bucket-type create mytype {props: {n_val: 4}}
         #  after the arguments were passed into the new shell during exec su
         #
-        # So this regex finds any '"', '{', or '}' and prepends with a '\'
-        ESCAPED_ARGS=`echo "$@" | sed -e 's/\([{}"]\)/\\\\\1/g'`
+        # So this regex finds any '(', ')', "'" , '"', '{', or '}' and prepends with a '\'
+        ESCAPED_ARGS=$(echo "$@" | sed -e "s/\([\\\(\\\){}\"']\)/\\\\\1/g")
```
Comment (kesslerm) [inline priv/base/env.sh:242]: `ESCAPED_ARGS=`echo "$@" | sed -e 's/\([\\\(\\\){}"\x27]\)/\\\\\1/g'`` should be equivalent.

The default Solaris shell only supports command substitution with backticks. The single quote character `'` can be expressed in sed expressions as `\x27`.

**Pull request merged**